### PR TITLE
Create employee documents accordion: upload, download, document progress and only accepting PDF uploads

### PR DESCRIPTION
**Pull request by LlewellynAnthony**
UploadDocumentEmployeeDocs
DocumentUpdatedEmployeeDocs
All_Documents_Uploaded
DownloadedDocument
IncorrectFileType
NoMissingInfo
EmployeeDocumentAccordionIpad
EmployeeDocumentsIpadExpanded
EmployeeDocumentsMobile
EmployeeDocsMobileExpanded

### CODE REVIEW

**Comment by dvanhout89** [comment]
Why is there a screenshot unrelated to HRIS?

**Comment by LlewellynAnthony** [comment]
> > UploadDocumentEmployeeDocs DocumentUpdatedEmployeeDocs All_Documents_Uploaded DownloadedDocument IncorrectFileType NoMissingInfo EmployeeDocumentAccordionIpad EmployeeDocumentsIpadExpanded EmployeeDocumentsMobile EmployeeDocsMobileExpanded
> 
> This doesnt look like the designs on figma. The buttons are not aligned.
> 
> image



> Why is there a screenshot unrelated to HRIS? image
> 
Proof that you can download PDFs from the accordion

**Comment by LlewellynAnthony** [comment]
> > UploadDocumentEmployeeDocs DocumentUpdatedEmployeeDocs All_Documents_Uploaded DownloadedDocument IncorrectFileType NoMissingInfo EmployeeDocumentAccordionIpad EmployeeDocumentsIpadExpanded EmployeeDocumentsMobile EmployeeDocsMobileExpanded
> 
> This doesnt look like the designs on figma. The buttons are not aligned.
> 
> image

Will fix tomorrow

File: src/app/components/hris/employees/employee-profile/accordions/accordion-documents/accordion-documents-starterkit/accordion-documents.component.ts [hunk 1]
```diff
@@ -42,6 +42,8 @@ export class AccordionDocumentsComponent {
   selectedFile !: File;
   roles: string[] = [];
   isLoadingUpload: boolean = false;
+  allowedTypes = ['application/pdf'];
+
```
Comment (madwizzard) [inline src/app/components/hris/employees/employee-profile/accordions/accordion-documents/accordion-documents-starterkit/accordion-documents.component.ts:46]:
```suggestion
```

File: src/app/components/hris/employees/employee-profile/accordions/accordion-employee-documents/accordion-employee-documents.component.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<div class="col-12" id="accordion-container">
+
```
Comment (madwizzard) [inline src/app/components/hris/employees/employee-profile/accordions/accordion-employee-documents/accordion-employee-documents.component.html:2]:
```suggestion
```

File: src/app/components/hris/employees/employee-profile/accordions/accordion-employee-documents/accordion-employee-documents.component.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    <mat-expansion-panel class="mt-3 rounded-3">
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          Employee Documents
+        </mat-panel-title>
+        <mat-panel-description *ngIf="documentFormProgress != 100" class="pe-0 pe-md-5">
+          <mat-icon class="mx-1">error</mat-icon><span class="d-none d-md-block d-lg-block"> Missing information</span>
+        </mat-panel-description>
+      </mat-expansion-panel-header>
+
```
Comment (madwizzard) [inline src/app/components/hris/employees/employee-profile/accordions/accordion-employee-documents/accordion-employee-documents.component.html:13]:
```suggestion
```

**Review by MathildaShongwe15 — APPROVED** [review]

**Review by Lucien-vanWyk — APPROVED** [review]

**Review by madwizzard — APPROVED** [review]

**Pull request merged**